lib/evss/power_of_attorney_verifier.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module EVSS
+  class PowerOfAttorneyVerifier
+    def initialize(user)
+      @user = user
+      @use_mock = Settings.evss.power_of_attorney_verifier || false
+      @veteran = Veteran.new(@user)
+    end
+
+    def verify(header)
```

Comment:
I would call this consumer_id or something like that rather than header is a little more ambiguous

Reply:
Changes have been made